code/Controllers/CMSMain.php

```diff
@@ -862,11 +862,16 @@ public function savetreenode($request)
             ->setBody(json_encode($statusUpdates));
     }
 
+    /**
+     * Whatever the current member has the permission to reorganise SiteTree objects.
+     * @return bool
+     */
     public function CanOrganiseSitetree()
     {
-        return !Permission::check('SITETREE_REORGANISE') && !Permission::check('ADMIN') ? false : true;
+        return Permission::check('SITETREE_REORGANISE');
```

Comment:
Is the admin check implicit?

Reply:
I see, this replaces #2033 and implements the suggested feedback from it

Reply:
Yes and there's an entry in the unit tests to check for both SITETREE_REORGANISE and ADMIN to be extra sure.

Reply:
Lovely!